packages/core-test-kit/test/test-stylable-core.spec.ts

```diff
@@ -70,7 +70,8 @@ describe(`testStylableCore()`, () => {
                     entries: [`/a.st.css`, `/b.st.css`],
                 }
             );
-
+            
+            // @ts-expect-error entry is not processed
```

Comment:
why is this needed? isn't the `Keys & Entries` used for `Sheets` intersect both entries and the passed files?
If it doesn't then I would ignore the entries option and use just the files, they should include all the entries anyway.